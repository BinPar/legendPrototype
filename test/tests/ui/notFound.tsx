import React from 'react';
import { mount } from 'enzyme';
import NotFound from '../../../pages/404';

/**
 * Checks that the 404 Not Found page renders correctly 
 */
test('Home Page', () => {

  // Renders the page
  const page = mount(<NotFound />);

  // We check that the content of the page is not changed
  expect(page).toMatchSnapshot();

});
