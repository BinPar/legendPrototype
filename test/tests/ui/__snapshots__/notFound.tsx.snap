// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page 1`] = `
<Component>
  <Head>
    <Component
      handleStateChange={null}
      inAmpMode={false}
      reduceComponentsToState={[Function]}
    />
  </Head>
  <h1>
    Error 404: This page could not be found
  </h1>
</Component>
`;
